browser/gui.cpp

```diff
@@ -17,6 +19,7 @@ int main() {
     http::Response response{};
     std::vector<dom::Node> dom{};
     std::string dom_str{};
+    std::string err_str{};
```

Comment:
Because clock is the oldest line and I haven't made any cleanup commits here yet. Won't touch it in an unrelated commit, so eventually I'll just make a cleanup pass. This entire gui client is just for testing the rest of the modules, so I'm not too worried about it.